Examples/HackerNewsClient/HackerNewsClient/Actions/GoBack.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  GoBack.swift
+//  HackerNewsClient
+//
+//  Created by Francesco Di Lorenzo on 15/11/2016.
+//  Copyright © 2016 BendingSpoons. All rights reserved.
+//
+
+import Katana
+
+struct GoBack: SyncAction {
+    typealias Payload = Void
+    var payload: Payload = ()
+    
+    static func updatedState(currentState: State, action: GoBack) -> State {
```

Comment:
In Katana 0.2.0 the action system is changed, I suggest to merge master in this branch and fix the issues